custom/icds_reports/static/js/directives/infants-weight-scale/infants-weight-scale.directive.js

```diff
@@ -14,7 +14,7 @@ function InfantsWeightScaleController($scope, $routeParams, $location, $filter,
     vm.data = {
         legendTitle: 'Percentage',
     };
-    vm.filters = ['gender', 'age', 'ageServiceDeliveryDashboard'];
```

Comment:
is vm.filters as global variable that it contained the 'ageServiceDeliveryDashboard' everywhere?

Reply:
In the ```vm.filters``` we have filters which we don't want to show in the report, so that's why we had this value in all report controller.